src/main/proto/netflix/titus/titus_agent_api.proto

```diff
@@ -32,6 +32,9 @@ enum InstanceLifecycleState {
 
     /// An agent instance is not running anymore.
     Stopped = 3;
+
+    /// An agent instance state is unknown.
+    InstanceStateUnknown = 4;
```

Comment:
Really minor, seems like calling it `Unknown` would be more consistent with the shorter names for the other enums. None of the others include the term `State`, which is also in the enum type name.